README.pt-br.md

```diff
@@ -1434,6 +1551,277 @@ configure :production, :test do
 end
 ```
 
+Você pode acessar essa opções através de `settings`:
+
+```ruby
+configure do
+  set :foo, 'bar'
+end
+
+get '/' do
+  settings.foo? # => true
+  settings.foo  # => 'bar'
+  ...
+end
+```
+
+### Configurando Proteção de Ataques
+
+Sinatra usa
+[Rack::Protection](https://github.com/sinatra/sinatra/tree/master/rack-protection#readme)
+para defender sua aplicação contra ataques comuns e oportunistas. Você pode
+facilmente desativar esse comportamento (o que abrirá sua aplicação para uma
+tonelada de vulnerabilidades):
+
+```ruby
+disable :protection
+```
+
+Para pular uma única camada de defesa, passe `protection` para as opções de um
```

Comment:
Here we're not talking about any hash, but the specific "options hash". So might be clearer to say:
```suggestion
Para pular uma única camada de defesa, passe `protection` para o hash de opções:
```